index.html

```diff
@@ -5347,7 +5419,7 @@ <h3 id="Elemental">Elemental</h3>
                <td><strong>Drop a large spiked rock on your foe.</strong>
                   <p>
                      Make a versatile MOD attack against a creature within 50ft. On a hit it suffers
-                     Heavy+MOD damage. This attack is a critical hit on <em>prone </em>targets.
+                     Medium+MOD damage. Deal an additional Heavy damage if the creature is <em>prone</em>.
```

Comment:
I think you actually asked this to potentially be nerfed lol - I have it noted that you said consistent heavy ranged damage just felt a bit eh. I wasn't sure either way but I tried it. If you have any other ideas for how to make it nicer or more fun I'll take em, otherwise I'm happy to revert.